src/julia-parser.scm

```diff
@@ -1027,7 +1027,7 @@
                                     (parse-string-literal s #t)))
                         (nxt (peek-token s))
                         (macname (symbol (string #\@ ex '_str)))
-                        (macstr (if (triplequote-string-literal? str) str (cadr str))))
```

Comment:
Is this a typo here?  This is changing it from picking up the 2nd element of `str` `(cadr str)`, to picking up the first `(car str)`.